.infrastructure/helm-chart/todoapp/templates/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{.Chart.Name}}-pv
+  namespace: {{.Values.todoapp.namespace}}
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Delete
+  accessModes:
+    - ReadWriteMany
+  capacity:
+    storage: {{.Values.todoapp.pvcStorageRequest}}
```

Comment:
Ensure that `{{.Values.todoapp.pvcStorageRequest}}` is defined in your `values.yaml` file for the `todoapp`. If it's not defined, it could lead to persistent volume configuration issues.